services/s3/src/main/java/software/amazon/awssdk/services/s3/auth/AwsChunkedEncodingInputStream.java

```diff
@@ -27,8 +27,8 @@
 import javax.crypto.spec.SecretKeySpec;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import software.amazon.awssdk.auth.signer.AbstractAwsSigner;
 import software.amazon.awssdk.auth.signer.SigningAlgorithm;
+import software.amazon.awssdk.auth.signer.internal.AbstractAwsSigner;
```

Comment:
Looks like there is just a constant we use in this class. Can we move that to keep this internal?

Reply:
Moved to AbstractAws4Signer